utils/constants.js

```diff
@@ -144,3 +144,30 @@ export const TEAM_LINKS = [
     },
   },
 ];
+
+export const UPCOMING = [
+  {
+    EventImage: "/Event Images/Android JAM.jpeg",
+    EventName: "Android Study JAM",
+    EventDate: "Dec 2, 2021",
+    EventDescript: "Info-session by Android facilitator",
+  }
+
+];
+
+
+export const EVENT_LIST = [
+  {
+    EventImage: "/Event Images/Git for Linux.jpeg",
+    EventName: "Git for Linux",
+    EventDate: "Oct 9, 2021",
+    EventDescript: "Virtual Hands-on Workshop",
+  },
+  {
+    EventImage: "/Event Images/30 Days of Cloud.jpeg",
+    EventName: "30 Days of Google Cloud",
+    EventDate: "Sep 25, 2021",
+    EventDescript: "Info-session by Cloud facilitator",
+  }
+  
+]
```

Comment:
can we create the upcoming list dynamically and have just a single source of events? else we would have to manually update the website every time an event is done. 
Hint: use numeric dates to compare easily.